openshift/e2e-tests-openshift.sh

```diff
@@ -171,14 +175,14 @@ function run_e2e_tests(){
     ./test/conformance/runtime/... \
     --kubeconfig "$KUBECONFIG" \
     --imagetemplate "$TEST_IMAGE_TEMPLATE" \
-    --resolvabledomain || failed=1
+    --resolvabledomain "$(ingress_class)" || failed=1
```

Comment:
Where does "ingress_class" come from here?

Reply:
It is from e2e-common.sh https://github.com/openshift/knative-serving/blob/release-next/test/e2e-common.sh#L392
The `openshift/e2e-tests-openshift.sh` explicitly includes the e2e-common.sh so it would be alright to use the func, I think.

Reply:
Gotcha, that works for me, yeah!